docs/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>YouTube Desktop</title>
+    <link rel=icon href=./favicon.png>
+    <link href="./reset.css" title='main' rel="stylesheet">
+    <link href="./styles.css" title='main' rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  </head>
+  <body>
+    <section class="top">
+      <img src="../app/assets/YouTube-logo.png" alt="">
+      <h1>YouTube for desktop</h1>
+
+
+
+
+
+      <p>A simple & beautiful app for YouTube's video platform.
+      </br>Browse without distractions on any OS</p>
+      <a href="#">
```

Comment:
let's make this link to the GitHub repo; so when the landing page is live, people can click the button and go somewhere.